### PR TITLE
Merge nf-core template 1.12

### PR DESCRIPTION
**Pull request by skrakau**
Merge template 1.12

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated
- [ ] If necessary, also make a PR on the [nf-core/mag branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/mag)

### CODE REVIEW

**Review by d4straub — APPROVED** [review]
Looks good!

**Comment by skrakau** [comment]
Hi @nf-core/core, the tests are hanging after merging the template and somehow "Run workflow tests (19.10.0) " is still expected. Not sure if I broke something. Does anyone know where this problem is coming from?

File: main.nf [hunk 1]
```diff
@@ -1523,18 +1523,18 @@ workflow.onComplete {
...
     // Render the HTML template
-    def hf = new File("$baseDir/assets/email_template.html")
+    def hf = new File("$projectDir/assets/email_template.html")
     def html_template = engine.createTemplate(hf).make(email_fields)
     def email_html = html_template.toString()
 
     // Render the sendmail template
-    def smail_fields = [ email: email_address, subject: subject, email_txt: email_txt, email_html: email_html, baseDir: "$baseDir", mqcFile: mqc_report, mqcMaxSize: params.max_multiqc_email_size.toBytes() ]
-    def sf = new File("$baseDir/assets/sendmail_template.txt")
+    def smail_fields = [ email: email_address, subject: subject, email_txt: email_txt, email_html: email_html, baseDir: "$projectDir", mqcFile: mqc_report, mqcMaxSize: params.max_multiqc_email_size.toBytes() ]
```
Comment (drpatelh) [inline main.nf:1536]:
```suggestion
    def smail_fields = [ email: email_address, subject: subject, email_txt: email_txt, email_html: email_html, projectDir: "$projectDir", mqcFile: mqc_report, mqcMaxSize: params.max_multiqc_email_size.toBytes() ]
```

**Review by drpatelh — APPROVED** [review]
Looks great @skrakau. The additional test is just a remnant of the previous tests that were being performed. We just need to remove that as a required test in `Settings > Branches > `dev`. Will do that now.

**Comment by skrakau** [comment]
Thanks @drpatelh !

**Pull request merged**